### DISCUSSION TITLE
Streaming issue

### DISCUSSION THREAD

**Discussion by Kath1292** [ExistentialAudio/BlackHole#808] [Q&A]
I am unable to stream songs from youtube or from any of my playlist only savan works,blackhole searches for song but after I click it doesn't play  when it's from  youtube or yt music and I have tried cleared cache, force stop or uninstall and install it doesn't work at all. Can anyone tell me what to do?

**Comment by devinroth** [comment]
Are you talking about the virtual audio driver? Because it sounds like you’re talking about something else. Probably getting confused with the other open source music player.

Devin

**Comment by Kath1292** [comment]
No I am talking about black hole only, like black hole gives the option to
listen audio from youtube right ,that's not working for me.

**Comment by devinroth** [comment]
What have you tried?

Devin

**Comment by Kath1292** [comment]
I have tried reinstalling app , clear cache and force stop on app ,and I
have even did software update on my android device there it seems that it
just cant play songs from yt only, jio saavan and radio works fine in app.
If u can help further please tell me how resolve it.

**Comment by devinroth** [comment]
I seriously think you’re talking about a different project. There is no cache to clear. There is no app. There is no android anything.

BlackHole is a macOS virtual Loopback driver.

Devin
